docker/genconfig/genconfig.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/yaml.v2"
+
+	"github.com/insolar/insolar/configuration"
+)
+
+/* Return env variable or default value, it env is not presented */
+func GetEnvDefault(key, defaultVal string) string {
+	val, ok := os.LookupEnv(key)
+	if !ok {
+		return defaultVal
+	}
+	return val
+}
+
+/* Return '<host>:<port>' on public docker interface */
+func getURI(port uint) string {
+	host := GetEnvDefault("IP", "127.0.0.1")
+	return fmt.Sprintf("%s:%d", host, port)
+}
+
+const (
+	defaultConfigPath            = "/etc/insolar/insolard.yaml"
+	defaultTranportListenPort    = 7900
+	defaultLogLevel              = "info"
+	defaultMetricsListenPort     = 8001
+	defaultRPCListenPort         = 18182
+	defaultInsgorundListenPort   = 18181
+	defaultAPIListenPort         = 19191
+	defaultJaegerEndpointPort    = 6831
+	defaultKeysPath              = "/etc/insolar/keys.json"
+	defaultCertPath              = "/etc/insolar/cert.json"
+	defaultDataDir               = "/var/lib/insolar/"
```

Comment:
FYI this code may become a problem, e.g. for Windows users. I don't believe we should hardcode patches like this, right in the code. Although I don't insist on changing this code right now, please try to avoid this style in the future.

Reply:
its only for linux docker containers